packages/volto/news/6306.bugfix

```diff
@@ -0,0 +1 @@
+Fixing disable new blocks
```

Comment:
Please see how to [write a good change log entry](https://6.docs.plone.org/contributing/index.html#write-a-good-change-log-entry).